src/index.html

```diff
@@ -7,6 +7,42 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div
+      class="calendar
+      calendar--start-day-sun
+      calendar--month-length-31"
+    >
+      <div class="calendar__day calendar__day--1"></div>
```

Comment:
dont need to use additional modificators, just use nth-child instead
```suggestion
      <div class="calendar__day"></div>
```